metals/src/main/scala/scala/meta/metals/Main.scala

```diff
@@ -17,14 +17,26 @@ import org.eclipse.lsp4j.jsonrpc.Launcher
 object Main {
   def main(args: Array[String]): Unit = {
     if (args.exists(Set("-v", "--version", "-version"))) {
-      val supportedScalaVersions =
-        BuildInfo.supportedScalaVersions.sorted.mkString(", ")
+      val supportedScala2Versions =
+        BuildInfo.supportedScala2Versions
+          .groupBy(_.substring(0, 4))
```

Comment:
```suggestion
          .groupBy(ScalaVersions.scalaBinaryVersionFromFullVersion)
```
?

Reply:
Ah of course, I totally forgot that I could use that. Switched.